index.js

```diff
@@ -43,14 +44,22 @@ module.exports = function (options) {
             //get extension - assume .js as default
             var ext = path.extname(file.path) || '.js';
             //check if parser for filetype exists
-            //TODO: perhaps just skip unsupported files
-            if (!leasot.isExtSupported(ext)) {
+            if (!leasot.isExtSupported(ext) && !options.skipUnsupported) {
                 var msg = ['File:', file.path, '- Extension', gutil.colors.red(ext),
                     'is not supported'
                 ].join(' ');
                 cb(new PluginError(pluginName, msg));
                 return;
             }
+            if (options.skipUnsupported) {
```

Comment:
If we got here it doesn't mean the file should be skipped, it might be a legit file